docs/concepts/workloads/pods/init-containers.md

```diff
@@ -17,7 +17,7 @@ scripts not present in an app image.
 
 {:toc}
 
-{% include 1-5-beta.md %}
+{% assign for_k8s_version="v1.5" %}{% include feature-state-beta.md %}
 
 **Once the feature exits beta, Init Containers will be specified in the PodSpec
```

Comment:
I was referring to "Once the feature exits." Another possibility is to use a background shade or color. Example: https://cloud.google.com/container-registry/docs/